addons/product/product_data.xml

```diff
@@ -188,14 +188,6 @@ parameter) will see those record just disappear.
             <field name="name">Public Pricelist</field>
         </record>
 
-        <!-- Price list item -->
-
-        <record id="item0" model="product.pricelist.item">
```

Comment:
This has already been solved by this revision https://github.com/odoo/odoo/commit/8528651d22db6b8ac47c87c772b1348bdd9d40fd